packages/core/src/Asset/util/metadatumToCip25.ts

```diff
@@ -85,28 +84,67 @@ const mapFile = (metadatum: Metadatum, assetId: Cardano.AssetId, logger: Logger)
 };
 
 /**
- * Also considers asset name encoded in utf8 within metadata valid
+ * Gets the `Map<AssetName, NFTMetadata>` relative to the given policyId from the given
+ * `Map<PolicyId, Map<AssetName, NFTMetadata>>`.
+ *
+ * The policyId in the `policy` Map can be encoded as per CIP-0025 v1 or v2 specifications.
+ *
+ * @param policy The `MetadatumMap` containing the NFT metadata for all the NFT assets
+ * @param policyId The policyId of the assent looking for
```

Comment:
```suggestion
 * @param policyId
```
This is self-descriptive. Is it legal based on our eslint rules to do this?

Reply:
Nope @rhyslbw , if I remove the `@param` description, eslint is not happy.
It seems we must provide at least a simple description.

Reply:
I think you can remove the entire @param line. Docs should be able to figure out that there is a param from typescript

Reply:
I've checked and @mkazlauskas 's suggestion seems to be a good solution.
Being the function not exported it is not included in the `typedoc` output.
I've tried running it exporting the function and the produced doc was fine even if without the parameter description

f9eceac70 should solve this